hoomd/md/methods.py

```diff
@@ -751,12 +750,12 @@ class Brownian(_Method):
             coefficient where :math:`d_i` is particle diameter.
             Defaults to None.
 
-        gamma (TypeParameter[``particle type``, `float`]): The drag coefficient
-            can be directly set instead of the ratio of particle diameter
-            (:math:`\gamma = \alpha d_i`). The type of ``gamma`` parameter is
-            either positive float or zero.
+        gamma (TypeParameter[ ``particle type``, `float` ]): The drag
+        coefficient can be directly set instead of the ratio of particle
+        diameter (:math:`\gamma = \alpha d_i`). The type of ``gamma`` parameter
+        is either positive float or zero.
```

Comment:
```suggestion
        gamma (TypeParameter[ ``particle type``, `float` ]): The drag
            coefficient can be directly set instead of the ratio of particle
            diameter (:math:`\gamma = \alpha d_i`). The type of ``gamma`` parameter
            is either positive float or zero.
```